release/rllib_tests/multi_gpu_learning_tests/multi_gpu_learning_tests.yaml

```diff
@@ -198,6 +199,9 @@ sac-repeat-after-me-env-continuous:
         initial_alpha: 0.001
         # Double batch size (2 GPUs).
         train_batch_size: 512
+        # start learning immediately 
```

Comment:
Can you explain why this fixed anything? :) Just curious. I would imagine the train batches to be very small in the beginning, then as there is nothing in the buffer and our rollout fragment length is 1.

Reply:
I have no idea what this env is. But one possible explanation is in this particular env, the initial random policy is not very good. By sampling until reaching that large threshold it makes the sampled batch super noisy and reliant on the old random policy. So if u start earlier you decrease the chance of sampling transitions from that old bad policy. This could potentially accelerate learning.